scripts/conda-requirements.yaml

```diff
@@ -0,0 +1,57 @@
+name: chipyard
+channels:
+  - conda-forge
+  - ucb-bar
+dependencies:
+  # firemarshal deps
+  - python=3.9
+  - rsync
+  - psutil
+  - doit=0.35.0
+  - gitpython
+  - humanfriendly
+  - e2fsprogs
+  - ctags
+  - bison
+  - flex
+  - expat
+  # current version of buildroot won't build with make 4.3 https://github.com/firesim/FireMarshal/issues/236
+  - make!=4.3
+  - pyyaml
+  - unzip
+  - readline
+  - coreutils
+  - lzop
+  - qemu # from ucb-bar
+
+  # doc requirements
+  - sphinx
+  - pygments
+  - sphinx-autobuild
+  - sphinx_rtd_theme
+  - docutils
+
+  # misc. c/c++ compilers + related
+  - gcc==10.* # pinned for libelf/libdwarf builds
+  - gxx==10.* # pinned for libelf/libdwarf builds
+  - conda-gcc-specs
+  - binutils
+
+  # rocket-chip deps
+  - sbt
+  - openjdk
+  - dtc
+  - verilator==4.034
+
+  # chipyard deps
+  - dromajo # from ucb-bar
```

Comment:
What is publishing dromajo?

Reply:
This repo: https://github.com/riscv-boom/dromajo. I can re-add this but I think so little people use Dromajo that I can just add a comment pointing to the feedstock (same thing for the QEMU version)

Reply:
Cool. I didn't catch that you added the conda scripts there too

Reply:
@abejgonzalez I think you're missing the `dromajo_cosim.h` header in your `dromajo` package.  At least the FireSim driver compile fails because it isn't in there.

Reply:
It should be added now.